docs/sources/flow/getting-started/opentelemetry-to-lgtm-stack.md

```diff
@@ -0,0 +1,308 @@
+---
+title: OpenTelemetry to Grafana stack
+weight: 350
+---
+
+# OpenTelemetry to Grafana stack
+
+In the [previous guide][], we learned how to ingest [OpenTelemetry][] data and export it to a different destination. This guide shows you how to send the ingested OpenTelemetry data to a Loki, Mimir, and Tempo stack.
```

Comment:
```suggestion
Grafana Agent Flow can be configured to collect [OpenTelemetry][]-compatible data and forward it to the Grafana stack

This topic describes how to:

* Configure Agent to send your data to Loki
* Configure agent to send your data to Tempo
* Configure Agent to store your data in Mimir
```

Suggesting some restructuring now that the document is more complete. This brings the intro in-line with the previous Getting started topics. We can shift the `previous guide` reference to the **Before you begin** section.